docs/book/src/user/quick-start.md

```diff
@@ -0,0 +1,79 @@
+# Quick Start
+
+In this tutorial we'll cover the basics of how to use Cluster API to create one or more Kubernetes clusters.
+
+## Prerequisites
+
+- Install and setup [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in your local environment.
+- A working Kubernetes cluster to be used as [management cluster](../reference/glossary.md#management-cluster).
+
+## Before we start
+
+#### Picking Infrastructure and Bootstrap Providers
+
+The [providers section](../reference/providers.md) offers a quick reference to some providers
+available and compatible with Cluster API _v1alpha2_.
+
+<aside class="note">
+<h1>Following Along vs Jumping Ahead</h1>
+
+If you're unfamiliar with the different providers types introduced in v1alpha2
+and how they interact with each other, please refer to the [user guide concepts].
+
+[user guide concepts]: ../user/concepts.md
+</aside>
+
+To proceed, we'll need to pick and choose a _Bootstrap_ and _Infrastructure_ provider.
+Once you've chosen which provider to use, we'll need to grab the [components]
+published as part of the release.
+
+> **NB**: Providers usually have their own getting started guide on how to generate additional resources,
+> configuration, credential management, etc. Refer to their documentation before proceeding.
+
+#### What if I don't have a management cluster?
+In some cases, you might not have a Kubernetes Cluster available to be used
+as management cluster for Cluster API resources.
+
+Services like Google Cloud GKE, Amazon EKS, Azure Kuberentes Service (AKS),
+or similar, are good examples of Kubernetes-as-a-service platforms
+that can be used for this purpose.
+
+##### Testing or development
+If you're looking for a temporary cluster for testing, development, or demos,
+we suggest you to use [kind].
+
+```bash
+GO111MODULE="on" go get sigs.k8s.io/kind@v0.5.1
+kind create cluster --name=clusterapi
+export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+```
+
+## Installation
+
+Each [release](https://github.com/kubernetes-sigs/cluster-api/releases) has a list of published assets.
+For the purpose of this tutorial, download the `cluster-api-components.yaml` file from the latest `v0.2.x` release.
+
+At this point, we should have all the required [components]:
+- Cluster API: `cluster-api-components.yaml`
+- Bootstrap: `bootstrap-components.yaml`
+- Infrastructure: `infrastructure-components.yaml`
+
+```bash
+kubectl create -f cluster-api-components.yaml
+kubectl create -f bootstrap-components.yaml
```

Comment:
I think at this point, It could become confusing where to get `bootstrap-components.yaml` and `infrastructure-components.yaml` from. Without explicitly calling out a provider for either is there a nice way to make reference of an out of box experience. Or would this be left up to the provider?